markdown/developers/developer_tutorials.md

```diff
@@ -7,6 +7,8 @@ subtitle: In-depth step-by-step guides for writing nf-core pipelines
 
 This page includes step-by-step guides, walkthroughs, and general long-form introductions to various aspects of nf-core.
 
+- [Tutorial: nf-core contributor overview](tutorials/nf_core_contributing_overview.md)
+  - An overview on contributing to nf-core, starting with installing nf-core tools, creating a new pipeline using the nf-core template, adding modules to it and releasing.
```

Comment:
```suggestion
  - An overview of how to contribute to nf-core, starting with installing nf-core tools, creating a new pipeline using the nf-core template, adding modules to a pipeline, and releasing a pipeline.
```

Reply:
If you don't like the changes feel free to ignore.

Reply:
Thanks for the input @christopher-hakkaart